src/EXTRA-FIX/fix_addtorque_group.cpp

```diff
@@ -36,10 +36,10 @@ enum{NONE,CONSTANT,EQUAL,ATOM};
 
 /* ---------------------------------------------------------------------- */
 
-FixAddTorque::FixAddTorque(LAMMPS *lmp, int narg, char **arg) :
+FixAddTorqueGroup::FixAddTorqueGroup(LAMMPS *lmp, int narg, char **arg) :
   Fix(lmp, narg, arg)
 {
-  if (narg != 6) error->all(FLERR,"Illegal fix addtorque command");
+  if (narg != 6) error->all(FLERR,"Illegal fix addtorque/group command");
 
```

Comment:
```suggestion

  if ((strcmp(style, "addtorque") == 0) && (comm->me == 0))
    error->warning(FLERR,"Fix addtorque was renamed to fix addtorque/group. Please update your input");
```

We should inform people with legacy inputs of the rename.

Reply:
Ditto here